openshift/topics/advanced_concepts.adoc

```diff
@@ -281,6 +281,12 @@ If successful, the previous command will return the following output:
 route "sso" patched
 ----
 
+[[sso-connecting-to-an-external-database]]
+==== Connecting to an external database
+
+{project_name} can be configured to connect to an external (to OpenShift cluster) database. In order to achieve this, you need to modify the `sso-{database name}` Endpoint object to point to proper address. The procedure has been described in https://docs.openshift.com/container-platform/3.4/dev_guide/expose_service/expose_internal_ip_service.html#manually-assign-ip-service[OpenShift manual].
```

Comment:
Please prefer using _`latest`_ version, when referring to OCP documentation, since that one will get automatically pointed to the next version, once it's available, e.g.:

```
https://docs.openshift.com/container-platform/latest/dev_guide/expose_service/expose_internal_ip_service.html#manually-assign-ip-service[OpenShift manual].
```

Besides that OCP v3.4 is an old version now.

Reply:
Fixed. Thanks!